data-plane/docker-compose.yml

```diff
@@ -15,6 +15,7 @@ services:
       - 6653:6653
       - 8181:8181
       - 8081:8080
+      - 8000:8000
```

Comment:
The Amlight container is supposed to be the OXPO, right? If that is the case, it means Kytos service should be running here. Thus, why you are exposing ports 8081 and 8000, if Kytos will only need 6653 and 8181 ?

Reply:
Thank you, my friend. I'm pleased to share that I've successfully integrated multiple components to create a comprehensive setup for our project. In particular, I've combined Kytos, the Kytos SDX Topology Napp, the Validator Flask module, and the SDX-LC Connexion module, all within a single Docker container designed for the Amlight environment.
I've also extended this approach to the Sax and Tenet environment. In this case, I've created another Dockers container that includes the same components, with the only difference being that it listens on distinct ports to accommodate the unique requirements of the Sax and Tenet environment.